stubs/SQLAlchemy/sqlalchemy/dialects/firebird/fdb.pyi

```diff
@@ -1,6 +1,6 @@
 from .kinterbasdb import FBDialect_kinterbasdb
 
-class FBDialect_fdb(FBDialect_kinterbasdb):
+class FBDialect_fdb(FBDialect_kinterbasdb):  # type: ignore[misc]
```

Comment:
Why do these need ignore comments?

Reply:
IIRC the derived classes don't implement all abstract functions, even though they are non-abstract. Ideally, we'd have a dedicated mypy error code for this problem, but I don't have the time to work on that. :(